tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/greater-less-than-operation.bal

```diff
@@ -24,6 +24,27 @@ function testFloatRanges(float a) returns (int) {
     return retunType;
 }
 
+function testByteComparison() {
+    byte lowValue = 0x20;
```

Comment:
Shall we same test with byte value with decimals.

Reply:
I ran the same test with byte value with decimals. It was fine. Do I need to add those lines here also.

Reply:
I mean lets keep the hexa decimal value as well.

something like

function testByteComparison() {
compareByteValues(0x20, 0x21);
compareByteValues(32, 45);
compare ByteValues(0x20, 35);
}

compare ByteValues(lowValue, highValue ) {// your compare test}
}

Reply:
I will look into this.